docs/script.js

```diff
@@ -30,7 +30,19 @@ const sampleCollection = [
 <label>
   <input type="radio" class="nes-radio" name="answer" />
   <span>No</span>
-</label>`,
+</label>
+
+<div style="background-color:#212529; padding: 1rem;">
+<label>
+  <input type="radio" class="nes-radio is-dark" name="answer-dark" checked />
+  <span>Yes</span>
+</label>
+
+<label>
+  <input type="radio" class="nes-radio is-dark" name="answer-dark" />
+  <span>No</span>
+</label>
+</div>`,
```

Comment:
the indentation is wrong here 🙈

Reply:
You're right, I didn't saw it, my bad. I'll fix it.

Reply:
![image](https://user-images.githubusercontent.com/22016005/53414194-eaebf080-39ac-11e9-936b-2ce8c2f2bce7.png)